File: easy/3856_trim_trailing_vowels/solution.py
class Solution:
    def trimTrailingVowels(self, s: str) -> str:
        vowels_set = set("aeiou")

        i = len(s) - 1
        while i != -1 and s[i] in vowels_set:
            i -= 1

        return s[: i + 1]

File: easy/3856_trim_trailing_vowels/solution.cpp
#include <string>
#include <unordered_set>

class Solution {
public:
  std::string trimTrailingVowels(std::string s) {
    std::unordered_set<char> vowelsSet = {'a', 'e', 'i', 'o', 'u'};

    while (!s.empty() && vowelsSet.find(s.back()) != vowelsSet.end())
      s.pop_back();

    return s;
  }
};
